hopsworks-web/yo/app/views/featurestore.html

```diff
@@ -610,5 +619,79 @@
                 </div>
             </div>
         </div>
+        <tour class="ng-scope" step="featurestoreCtrl.tourService.currentStep_TourNine"
+              post-tour="closeTour()"
+              ng-if="featurestoreCtrl.tourService.activeTour === 'featurestore'
+          && (featurestoreCtrl.tourService.informAndTips
+              || featurestoreCtrl.tourService.tipsOnly) && !featurestoreCtrl.loading && !projectCtrl.loading">
+            <virtual-step
+                    ng-if="featurestoreCtrl.tourService.activeTour === 'featurestore'"
+                    class="ng-scope"
+                    tourtip="This is the feature store registry where you can list and search for features/feature groups
+                    and training datasets in the feature store.
+                    You can also edit metadata of features/feature groups and training datasets from here"
+                    tourtip-step="0"
+                    tourtip-element="#featurestoreTab"
+                    tourtip-placement="bottom"
+                    tourtip-next-label="Next"
+                    on-proceed=""
+            ></virtual-step>
+            <virtual-step
+                    ng-if="featurestoreCtrl.tourService.activeTour === 'featurestore'"
+                    class="ng-scope"
+                    tourtip="Click here to select a feature store. Each project with the feature store service gets its own feature store.
```

Comment:
@Limmen `Click here to select the project's feature store. A feature store can also be shared across projects. You can find a feature store that has been shared with your project in the drop-down list here`

Reply:
Done